terraform/snowflake/modules/elt/roles.tf

```diff
@@ -44,6 +44,24 @@ resource "snowflake_account_role" "logger" {
   comment  = "Permissions to read the SNOWFLAKE metadatabase for logging purposes"
 }
 
+# New Streamlit Roles - one for each database
+resource "snowflake_account_role" "streamlit_raw" {
+  provider = snowflake.useradmin
+  name     = "${module.raw.name}_${var.environment}_STREAMLIT"
+  comment  = "Permissions to create Streamlit applications and stages in the ${module.raw.name} database for the ${var.environment} environment."
+}
```

Comment:
I don't think we need to have a streamlit role for `RAW` or `TRANSFORM`: the `ANALYTICS` database is meant to be the presentation layer, and people creating streamlit apps in the others wouldn't be following the intent of the RBAC structure